plugins/webp-uploads/helper.php

```diff
@@ -144,7 +145,7 @@ function webp_uploads_generate_additional_image_source( int $attachment_id, stri
 		return new WP_Error( 'image_mime_type_not_supported', __( 'The provided mime type is not supported.', 'webp-uploads' ) );
 	}
 
-	$image_path = wp_get_original_image_path( $attachment_id );
+	$image_path = 'application/pdf' !== get_post_mime_type( $attachment_id ) ? wp_get_original_image_path( $attachment_id ) : get_attached_file( $attachment_id );
```

Comment:
Minor point: This is easier to read and it would facilitate ensuring test coverage is present for both code paths. Also, in the future maybe there will be more such special cases.

```suggestion
	if ( 'application/pdf' === get_post_mime_type( $attachment_id ) {
		$image_path = get_attached_file( $attachment_id );	
	} else {
		$image_path = wp_get_original_image_path( $attachment_id );	}
```